### PR TITLE
Fix warnings at the top 

### PR DESCRIPTION
**Pull request by mluu12**
Closes https://github.com/influxdata/docs-v2/issues/3097

Fixed the warnings in Cloud's "Experimental" doc so it doesn't sound as aggressive. 

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud/reference/flux/stdlib/experimental/_index.md [hunk 1]
```diff
@@ -16,15 +16,14 @@ The Flux Experimental package includes experimental functions that perform vario
...
-Experimental functions are subject to change and are **not recommended for production use**.
-At any time, experimental functions and packages may:
+**Experimental functions are subject to change**
+
+Please note that experimental functions and packages may: 
 
 - be moved or promoted to a permanent location
 - undergo API changes
 - stop working with no planned fixes
 - be removed without warning nor published explanation
```
Comment (kelseiv) [inline content/influxdb/cloud/reference/flux/stdlib/experimental/_index.md:26]:
```suggestion
- be removed without warning or explanation
```

**Review by kelseiv — APPROVED** [review]
@mluu12 let's update the related oss doc. 
cc @sanderson in case this change affects the Flux restructure branch

**Comment by sanderson** [comment]
Thanks for this @mluu12. I'm closing this PR in favor of #3128 which ports these changes to the Flux restructure branch. That docs that this PR updates will be removed with the restructure.

**Comment by kelseiv** [comment]
@sanderson - thanks for updating the Flux restructure branch! 
Any reason/conflict why @mluu12 shouldn't push these changes out anyway, so we can close the priority issue?

**Comment by sanderson** [comment]
@kelseiv I'm planning on pushing my change out tomorrow with the `flux-restructure` branch. These changes will cause a merge conflict with my branch since I'm removing all the files that are updated by this PR.

**Pull request closed without merging**